The Netwinder machines with Cyber2010 crash badly when starting Xserver.
The workaround is to disable pci burst option for this revision of video chip.

Signed-off-by: Woody Suwalski <woodys@xandros.com>

---

--- a/drivers/video/cyber2000fb.c	2006-04-21 12:11:45.000000000 -0400
+++ b/drivers/video/cyber2000fb.c	2006-04-26 16:29:44.000000000 -0400
@@ -1541,6 +1541,12 @@ static int cyberpro_pci_enable_mmio(stru
 	/*
 	 * Allow the CyberPro to accept PCI burst accesses
 	 */
+	if (cfb->id == ID_CYBERPRO_2010)
+	{
+		printk(KERN_INFO "%s: NOT enabling PCI bursts\n", cfb->fb.fix.id);
+	}
+	else
+	{
 	val = cyber2000_grphr(EXT_BUS_CTL, cfb);
 	if (!(val & EXT_BUS_CTL_PCIBURST_WRITE)) {
 		printk(KERN_INFO "%s: enabling PCI bursts\n", cfb->fb.fix.id);
@@ -1552,6 +1558,7 @@ static int cyberpro_pci_enable_mmio(stru
 
 		cyber2000_grphw(EXT_BUS_CTL, val, cfb);
 	}
+	}
 
 	return 0;
 }
